src/components/Admin/Home/AdminHome.tsx

```diff
@@ -16,7 +16,7 @@ type Props = {
   envVars: any,
 };
 
-const AdminHome = (props: Props): JSX.Element => {
+export const AdminHome = (props: Props): JSX.Element => {
```

Comment:
```suggestion
export const AdminHome:FC<Props> = (props: Props) => {
```
こちらの方が良さそうです
参考：https://qiita.com/Takepepe/items/0bbe4ab6ff4858a4e186#-winners

Reply:
修正しました